app/frontend/src/components/ApplicationList.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-container>
+    <h2>My Applications</h2>
+    <p>*Submit the Request Account form to add a new application</p>
```

Comment:
That asterisk looks a bit strange - may need to confirm with Mina what intent for this is (feels like she wanted it to be a bullet point?)

Reply:
Confirmed an * for now. Might revisit making it an info icon later on. But intended as is for now.